tools/diffscripts/README.md

```diff
@@ -0,0 +1,208 @@
+# CSharp SDK Diff Automation
+
+## Overview
+During the release process we need to examine the changes to the SDK surface to make sure we're not making too many changes, breaking or otherwise. This tool will automate the process of generating the mark down files so we do not need to use the GUI version which is time consuming.
```

Comment:
to make sure we're not making ~too many~ breaking changes ~,breaking or otherwise~.?
non-breaking changes should be ok, no matter how many :)

Reply:
True 😄 not sure what I was getting at (documentation fatigue probably)

Reply:
I'd say breaking or unintended changes